src/components/ArticlesCard/ArticleCard.jsx

```diff
@@ -1,17 +1,45 @@
-function ArticleCard({article}){
-    return (
-       <>
-        <article >
-            <li><img 
+import { useNavigate } from "react-router-dom";
+
+function ArticleCard({ article }) {
+  const navigate = useNavigate();
+  return (
+    <>
+      <article className="article" key={article.article_id}>
+        <header className="article-header">
+          <li>
+            <h3 className="hover-title">
+              <a
+                onClick={() => {
+                  navigate(`/articles/${article.article_id}`);
+                }}
+                role="link"
+              >
+                {article.title}
+              </a>
+            </h3>
+          </li>
+
+          <li>
+            <p>{article.author}</p>
+          </li>
+        </header>
+
+        <li>
+          <img
+            onClick={() => {
```

Comment:
if you wrap the whole thing in Link tag that will make all child element go to that link on click.  Will remove the need to have two onClick event listeners.